README.md

```diff
@@ -42,6 +42,22 @@ steps:
             role_arn: "arn:aws:iam::0015615400570:role/demo"
 ```
 
+It's also possible to assume a role using OIDC. This will call `buildkite-agent oidc request-token --audience sts.amazonaws.com`
+and exchange the resulting token for AWS credentials which are then used for the ECR login
+
+```yml
+steps:
+  - command: ./run_build.sh
+    plugins:
+      - ecr#v2.7.0:
+          login: true
+          account-ids: "0015615400570"
+          region: "ap-southeast-2"
+          assume_role:
+            role_arn: "arn:aws:iam::0015615400570:role/demo"
```

Comment:
Underscore `role_arn` seems at odds with other params.
Should it be `role-arn`?

Reply:
(Seems like the existing examples are mixed between `-` and `_`, and I guess both work)